script.js

```diff
@@ -15,4 +15,179 @@ hamburger.addEventListener('click', openWindow);
 closeIcon.addEventListener('click', closeWindow);
 for (let i = 0; i < linksPopUp.length; i += 1) {
   linksPopUp[i].addEventListener('click', closeWindow);
+}
+
+// PopUp Window
+const projects = [
+  {
+    name: 'Multi-Post Stories',
```

Comment:
- [ ]please kindly reduce the gaps between the tech buttons on the top left of the desktop popup, and that of the mobile version, so it matches the original design.
yours
![Screenshot (1277)](https://github.com/JhonatanErosa/Portfolio/assets/65298048/ce79ac6b-8312-499d-bd15-4ba8dd40c87e)
![Screenshot (1279)](https://github.com/JhonatanErosa/Portfolio/assets/65298048/214bbb16-ab53-4f19-a069-5f7399079b73)
figma
![Screenshot (1278)](https://github.com/JhonatanErosa/Portfolio/assets/65298048/a9dceab9-2cfd-4963-88a2-cb00643a7d23)
![Screenshot (1280)](https://github.com/JhonatanErosa/Portfolio/assets/65298048/d20cacb0-5cb5-4293-bbcf-b4e93d4c44c3)